matrix/common/aws/dynamo_lock.py

```diff
@@ -1,23 +1,22 @@
-"""TODO TEST AND/OR REFACTOR THIS CLASS OUT OF IMPLEMENTATION.
+"""TODO: REFACTOR THIS CLASS OUT OF IMPLEMENTATION.
```

Comment:
We should just remove this todo out of the code. We can create a spike ticket to investigate options that don't involve dynamo lock

Reply:
Yes please. Let's please avoid **TODO** in code and create tracking issues as necessary for visibility.

Reply:
Cool, created https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/matrix-service/159